pom.xml

```diff
@@ -31,7 +25,7 @@
       <alfresco.client.war.version>${alfresco-community.version}</alfresco.client.war.version>
       <alfresco.google.config.version>1.0.0-SNAPSHOT</alfresco.google.config.version>
       <alfresco.google.config.classifier>community</alfresco.google.config.classifier>
-      <compiler-plugin.version>3.1</compiler-plugin.version>
+      <compiler-plugin.version>3.8.0</compiler-plugin.version>
       <java.version>1.7</java.version>
```

Comment:
Will we leave as 1.7 or could / should this be 1.8 (for 3.0.N targeting ACS 5.2.x) ?

Reply:
We can leave 1.7, I think.

Reply:
If we need to do 3.0.4.x hot fix in the future then we could update at that point. For ref:

https://docs.alfresco.com/5.2/concepts/supported-platforms-ACS.html
.../Repository/repo/blob/support/SP/5.2.N/pom.xml#L664